src/database/models/reportQueries.js

```diff
@@ -51,7 +50,7 @@ module.exports = (sequelize, DataTypes) => {
 	// Define associations
 	ReportQuery.associate = (models) => {
 		ReportQuery.belongsTo(models.Report, {
-			foreignKey: 'report_id',
+			foreignKey: 'report_code',
```

Comment:
@sumanvpacewisdom if ur reference in the query, we need to check it works in citus or not